Dockerfile

```diff
@@ -12,10 +20,21 @@ COPY . .
 ENV THREAD_COUNT 20
 RUN set -e \
  && echo '#!/bin/bash -e' > /runserver \
- && echo 'waitress-serve --threads $THREAD_COUNT --port 80 snoop.wsgi:application' >> /runserver \
+ && echo 'waitress-serve --threads $THREAD_COUNT --port 8080 snoop.wsgi:application' >> /runserver \
  && chmod +x /runserver
 
+RUN chown -R $UID:$GID /runserver
+RUN chown -R $UID:$GID /opt/magic-definitions
+RUN chown -R $UID:$GID /opt/libpst
+
+ENV DATA_DIR "/opt/hoover/snoop"
```

Comment:
This is only a mount point when mounted from liquid with `mount_local_repos`, a use-case which you shouldn't care about here.

The mount point we care about here is `/opt/hoover/snoop/blobs` which is filled with directories, one for each collection, with blobs. Call it "$BLOBS_DIR"